apps/evm/lib/evm/gas.ex

```diff
@@ -32,8 +32,6 @@ defmodule EVM.Gas do
   @g_sset 20_000
   # Paid for an SSTORE operation when the storage value’s zeroness remains unchanged or is set to zero.
   @g_sreset 5000
-  # Amount of gas to pay for a SELFDESTRUCT operation.
-  @g_selfdestruct 5000
```

Comment:
Why are we deleting `@g_selfdestruct` and `g_txcreate`? Are they defined elsewhere?

Reply:
@germsvel yes, @g_selfdestruct is defined in `Refunds` module, @g_txcreate is defined is `CreateContract` module